R/zchunk_L111.nonghg_en_R_S_T_Y.R

```diff
@@ -1,110 +1,303 @@
-    # There's also a `same_precursors_as(x)` you can use
-    # If no precursors (very rare) don't call `add_precursor` at all
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+
+    L111.nonghg_tg_R_en_S_F_Yh %>%
+      add_title("Non-ghg emission totals by GCAM sector, fuel, technology, and driver type for EDGAR historical years.") %>%
+      add_units("Tg") %>%
+      add_comments("First, computed unscaled non-ghg emissions by country and technology.") %>%
+      add_comments("Then, computed EDGAR emissions by region and sector.") %>%
+      add_comments("Then, scaled EPA emissions by tech to match EDGAR totals.") %>%
+      add_comments("Then, computed international shipping and international aviation emissions.") %>%
+      add_comments("Finally, calculated non-ghg emission totals by GCAM sector, fuel, technology, and driver type for EDGAR historical years.") %>%
       add_legacy_name("L111.nonghg_tg_R_en_S_F_Yh") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
-      # typical flags, but there are others--see `constants.R`
+      add_precursors("common/iso_GCAM_regID",
+                     "emissions/EDGAR/EDGAR_sector",
+                     "emissions/mappings/EPA_tech",
+                     "emissions/mappings/GCAM_sector_tech",
+                     "L101.in_EJ_R_en_Si_F_Yh",
+                     "L101.so2_tgej_USA_en_Sepa_F_Yh",
+                     "L101.co_tgej_USA_en_Sepa_F_Yh",
+                     "L101.nox_tgej_USA_en_Sepa_F_Yh",
+                     "L101.voc_tgej_USA_en_Sepa_F_Yh",
+                     "L101.nh3_tgej_USA_en_Sepa_F_Yh",
+                     "emissions/EDGAR/EDGAR_SO2",
+                     "emissions/EDGAR/EDGAR_CO",
+                     "emissions/EDGAR/EDGAR_NOx",
+                     "emissions/EDGAR/EDGAR_NMVOC",
+                     "emissions/EDGAR/EDGAR_NH3") %>%
       add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
       L111.nonghg_tg_R_en_S_F_Yh
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+
+    L111.nonghg_tgej_R_en_S_F_Yh %>%
+      add_title("Non-ghg emission total shares by GCAM sector, fuel, technology, and driver type for EDGAR historical years.") %>%
+      add_units("Tg/EJ") %>%
+      add_comments("Used non-ghg emission totals by GCAM sector, fuel, technology, and driver type for EDGAR historical years to derive emission shares.") %>%
       add_legacy_name("L111.nonghg_tgej_R_en_S_F_Yh") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
-      # typical flags, but there are others--see `constants.R`
+      add_precursors("common/iso_GCAM_regID",
```

Comment:
Is it correct that _all_ precursors flow into _both_ outputs? Please check. Also, if yes, use `same_precursors_as`.